src/Controller/api/connector.c

```diff
@@ -168,3 +168,15 @@ int wb_connector_get_presence(WbDeviceTag tag) {
   robot_mutex_unlock_step();
   return result;
 }
+
+bool wb_connector_is_locked(WbDeviceTag tag) {
+  bool result = false;
+  robot_mutex_lock_step();
+  Connector *con = connector_get_struct(tag);
+  if (con)
+    result = con->is_locked;
+  else
+    fprintf(stderr, "Error: %s(): invalid device tag.\n", __FUNCTION__);
```

Comment:
```suggestion
  bool result;
  robot_mutex_lock_step();
  Connector *con = connector_get_struct(tag);
  if (con)
    result = con->is_locked;
  else {
    result = false;
    fprintf(stderr, "Error: %s(): invalid device tag.\n", __FUNCTION__);
  }
```
This is slightly more efficient and more readable to me.